cypress/e2e/Frontend-FX/FHG-929-FX-filter-distance.cy.js

```diff
@@ -6,130 +6,92 @@ describe('FHG-929-FX-filter-distance.spec',function(){
         cy.ServiceFilterPage();
         cy.familyHubDetails();
         cy.serviceDetails();
-        // both filters unchecked
-        cy.get('.govuk-grid-column-two-thirds').contains('Free').should('exist')
-        cy.get('.govuk-grid-column-two-thirds').contains('£').should('exist')
+
         // no filters tagged
         cy.get('.moj-filter__selected').contains('20 miles').should('exist')
-        // apply free filter    
+        // clear filter
         cy.clearFilters()
-        cy.get('.govuk-grid-column-two-thirds').contains('Free').should('exist')
-        cy.get('.govuk-grid-column-two-thirds').contains('£').should('exist')
+
         // no filters tagged
         cy.get('.moj-filter__selected').contains('20 miles').should('not.exist')
     });
     it('AC 2 - filter by 1 miles distance ',function(){
         cy.visit('/PostcodeSearch')
         cy.searchHubsPage()
         cy.searchbypostcode('m6 5uj')
-        // cy.ServiceFilterPage();
-        // cy.familyHubDetails();
-        // cy.serviceDetails();
-        // both filters unchecked
-        cy.get('.govuk-grid-column-two-thirds').contains('Free').should('exist')
-        cy.get('.govuk-grid-column-two-thirds').contains('£').should('exist')
+        cy.ServiceFilterPage();
+        cy.familyHubDetails();
+        cy.serviceDetails();
```

Comment:
optional nitpick: should we update the cashing of `SearchFilterPage` to match the others? (e.g. `familyHubDetails()`)

Reply:
Done and tests rerun to make sure no breakages